server/service/src/requisition/request_requisition/generate.rs

```diff
@@ -92,3 +95,36 @@ pub fn generate_requisition_lines(
 
     Ok(result)
 }
+
+pub(crate) fn generate_program_indicator_values(
+    store_id: &str,
+    period_id: &str,
+    program_indicators: Vec<ProgramIndicator>,
+) -> Vec<IndicatorValueRow> {
+    let mut indicator_values = vec![];
```

Comment:
I had it abstracted initially, but for request requisitions it is slightly different because there is no 'other_party_id'.
I thought it might be confusing to use that function but pass in store_id twice, once as other_party_id, so moved it back to two separate functions.

 Is it still worth abstracting into one?

Reply:
You can add a check for type... so you know what id to use

Reply:
I think you missed this comment @fergie-nz

Reply:
thank you! I did